dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -100,6 +101,10 @@ public final class ZenodoHelper {
     private static String zenodoUrl;
     private static String zenodoClientID;
     private static String zenodoClientSecret;
+    /**
+     * Comma-separated list of GitHub organization names and/or GitHub repo names, e.g., &qout;broadinstitute,DataBiosphere/topmed-workflow-variant-calling&quot;
+     */
+    private static List<String> autoGeneratedDoiAllowList;
```

Comment:
Could make this a Set.   We probably will never enough whitelisted orgs/repos that it matters performance-wise, but unless there's a compelling reason to keep it a List, might as well...